README.md

```diff
@@ -44,23 +44,37 @@ For development purposes, you can set up `100eyes` using Docker. To perform the
     You can start all containers with a single command. `100eyes` can then be viewed in your browser at http://localhost:3000.
 
     ```bash
-    $ docker-compose up
+    $ docker compose up
     ```
 
-5. #### Seed database
+5. #### Create database
+
+   The database is not created in the steps above because it is a one-time task, whereas building and starting the services should be reproducible. Create the database with:
+
+   ```bash
+   $ docker exec 100eyes-app-1 rails db:create
+   ```
+
+6. #### Run the migrations
+
+   ```bash
+   $ docker exec 100eyes-app-1 rails db:migrate
+   ```
+
+7. #### Seed database
```

Comment:
I think creating the database, running migration and seeds would need to go before running `docker compose up`, i.e.

1. Clone repo
2. Building container
3. Configuration
4. Create, migrate, seed
5. Start containers

Instead of running `rails db:create`, `rails db:migrate`, and `rails db:seed`, you should also be able to run `rails db:setup`. It’s basically the same, but I think it doesn’t run all migration, instead loads the schema from `schema.rb`, so it’ll be a little faster when setting up a dev environment from scratch.

Reply:
I thought so as well, but when I attempted to do that, I got an error. I'll see if I can reproduce it, maybe it was due to running the command with `docker` instead of `docker compose`.

Cool, yeah I had forgotten about the `db:setup` command, but that's a nice shortcut.

Reply:
Okay, cool! If I find the time, I’ll also try going through the setup instructions with fresh containers/volumes.

Reply:
the reason it was failing is because I ran the command with `exec`, but without the containers running it needs to be run with `run`. See 05a8ac68